css/styles.css

```
@@ -97,6 +97,11 @@ form {
   border-color: var(--secondary-color);
 }
 
+.form--blocked input#siteUrl,
+.form--blocked .card-type {
+  opacity: 30%;
+}
```

Since we're not blocking the whole form, I'd mark the fields as blocked instead. 
Doing it this way, there si no need for additional classes.
```suggestion
.form__input:disabled ,
.card-type:disabled {
  opacity: 30%;
}
```